torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2706,10 +2709,17 @@ def _register_post_forward_hooks(self):
         self._post_forward_handles.clear()
         for module in self.module.modules():
             module_param_handles = self._module_to_handles[module]
+            # reshard following handles: 1. the handle is in
```

Comment:
nit:
```suggestion
            # Reshard the handles satisfying:
            # 1. the handle is in `module_param_handles` and
            # 2. the handle does not contain parameters in the parent module of `module`
```
(fyi: I single backtick for non-docstring comments since it saves space and does not render. I wanted to specify `module` instead of "the current module" to be clear that the current module refers to the exact variable `module` in the for loop.)